docs/UserGuide.md

```diff
@@ -150,9 +139,9 @@ Format: `exit`
 Action | Format, Examples
 --------|------------------
 **Help** | `help`
-**Add** | `add c/CATEGORY  n/EXPENSE_NAME a/EXPENSE_AMT`
-**Remove** | `remove ENTRY_NUM`<br>`remove n/EXPENSE_NAME`
-**Edit** | `edit ENTRY_NUM n/EXPENSE_NAME a/MONEY_AMT`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`
-**List** | `list CATEGORY`
+**Add** | `add c/CATEGORY d/DESCRIPTION a/AMOUNT [t/TAG]`
+**Delete** | `delete ENTRY_INDEX c/CATEGORY`
 **Exit** | `exit`
+
```

Comment:
This is still Work In Progress right? Since by wednesday there will be new commands

Reply:
Yup will add once they're implemented! Some are already inside but commented out

Reply:
oh ok roger